app/src/androidTest/java/com/sdp/movemeet/view/activity/UploadActivityActivityTest.java

```diff
@@ -89,8 +90,7 @@
     @Before
     public void setup() {
         // Disable navigation for tests
-        UploadActivityActivity.enableNav = false;
-        MainActivity.enableNav = false;
+        Navigation.profileField = false;
```

Comment:
So, as discussed yesterday evening, the purpose of this Navigation.profileField Boolean is to be able to navigate freely inside the app without having to consider a specific user and hence to have to print his information in the side panel (which makes it possible to avoid that tests do not pass). Is this true explained this way?